apps/api/src/modules/auth/auth.service.ts

```diff
@@ -28,11 +32,29 @@ export class AuthService {
     return null;
   }
 
-  async login(loginUserInput): Promise<LoginResponse> {
+  async login(loginUserInput: LoginUserInput): Promise<LoginResponse> {
     const token = this.configService.getOrThrow<string>('SERVER_API_KEY');
```

Comment:
This is deprecated now right? Since we are getting the server API keys from DB now.

We should remove this and just maintain the `tokenList`. Even for non-admin users, we will have only one token in that array.

Need to cross-check this with @osm-vishnukyatannawar to ensure we don't break the current Portal completely.

Reply:
In the apiGuards this is still the only source of verifying tokens.
I can remove it from here

Reply:
User IDs are associated to Application IDs, and then Application IDs are associated with Server API Keys right? So we should be able to fetch only the Server API keys that the particular user has access to.
In case of admin, we return all active keys.

Reply:
Ok that is doable, I assume that this change is only for LoginResponse, and that we are not disturbing code in apiGuards

Reply:
Consider that we will be getting rid of the Server API Key from env too. If it can be done, please update the API Guard as well so that it checks for the API key in the database, not from the env.

Reply:
Changed this